### ISSUE TITLE
[CLI] Extra newline gets printed out when compressing multiple files

### ISSUE THREAD

**Issue by terrelln** [facebook/zstd#2855]
**Describe the bug**
An extra newline gets printed out when compressing multiple files and piping to stdout

**To Reproduce**

```
> touch f1 f2
> zstd f1 f2 -c | wc -c

26
```

**Expected behavior**
No extra newline:

```
> touch f1 f2
> zstd f1 f2 -c | wc -c
26
```

**Comment by 15596858998** [comment]
I think this problem is caused by the implementation of the "-c" command. Even if I only use "-c", there will be an extra line break.
![image](https://user-images.githubusercontent.com/33825201/141708554-9ea6e34b-03f4-47af-8e86-57ac43c58762.png)

**Comment by Cyan4973** [comment]
Fixed in #2876
